From 7c7419c6ad30e987613b4618dae55b2b1db84839 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 4 Dec 2020 13:56:47 -0800
Subject: [PATCH] Volgograd change likely December 27, not 20

(Thanks to Stepan Golosunov.)
* NEWS: Mention this.
* europe (Europe/Volgograd): Change predicted transition from +04
to +03 from Dec. 20 to Dec. 27.
---
 NEWS   |  2 +-
 europe | 12 ++++++------
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/NEWS b/NEWS
index b56db99..76a5417 100644
--- a/NEWS
+++ b/NEWS
@@ -5,7 +5,7 @@ Unreleased, experimental changes
   Changes to future timestamps
 
     Volgograd changes time zone from +04 to +03 on 2020-12-20 at 02:00.
-    (Thanks to Alexander Krivenyshev.)
+    (Thanks to Alexander Krivenyshev and Stepan Golosunov.)
 
   Changes to past timestamps
 
diff --git a/europe b/europe
index 3a8a4e6..fd9e85b 100644
--- a/europe
+++ b/europe
@@ -2894,12 +2894,12 @@ Zone Europe/Astrakhan	 3:12:12 -	LMT	1924 May
 
 # From Alexander Krivenyshev (2020-11-27):
 # The State Duma approved (Nov 24, 2020) the transition of the Volgograd
-# region to the Moscow time zone....  If approved by the the Federation
-# Council and the President, this Federal Law 1012130-7 will enter into force
-# on December 20, 2020....
+# region to the Moscow time zone....
 # https://sozd.duma.gov.ru/bill/1012130-7
-# http://docs.cntd.ru/document/566418247
-# https://gorvesti.ru/society/gosduma-progolosovala-za-vozvrat-volgogradskoy-oblasti-k-moskovskomu-vremeni-95877.html
+#
+# From Stepan Golosunov (2020-12-05):
+# Currently proposed text for the second reading (expected on December 8) ...
+# changes the date to December 27. https://v1.ru/text/gorod/2020/12/04/69601031/
 
 Zone Europe/Volgograd	 2:57:40 -	LMT	1920 Jan  3
 			 3:00	-	+03	1930 Jun 21
@@ -2910,7 +2910,7 @@ Zone Europe/Volgograd	 2:57:40 -	LMT	1920 Jan  3
 			 3:00	Russia	+03/+04	2011 Mar 27  2:00s
 			 4:00	-	+04	2014 Oct 26  2:00s
 			 3:00	-	+03	2018 Oct 28  2:00s
-			 4:00	-	+04	2020 Dec 20  2:00s
+			 4:00	-	+04	2020 Dec 27  2:00s
 			 3:00	-	+03
 
 # From Paul Eggert (2016-11-11):
-- 
1.8.3.1

